src/gen/api/accountingApi.ts

```diff
@@ -8628,9 +8779,10 @@ export class AccountingApi {
      * @param page e.g. page&#x3D;1 – Up to 100 invoices will be returned in a single API call with line items shown for each invoice
      * @param includeArchived e.g. includeArchived&#x3D;true - Invoices with a status of ARCHIVED will be included in the response
      * @param createdByMyApp When set to true you\&#39;ll only retrieve Invoices created by your app
+     * @param summaryOnly Use summaryOnly&#x3D;true in GET Contacts and Invoices endpoint to retrieve a smaller version of the response object. This returns only lightweight fields, excluding computation-heavy fields from the response, making the API calls quick and efficient.
```

Comment:
shouldn't new params be added last to avoid changing the order of existing params that devs might already be passing as arguments?

Reply:
after unitdp?